packages/terra-layout/docs/README.md

```diff
@@ -14,30 +14,3 @@ To facilitate communication between the Layout and its content, the Layout provi
 * `menuIsOpen` - A boolean flag indicating whether or not the menu is currently displayed.
 * `togglePin` - A function that will toggle the current menu's pinned state. This function is provied only to the `menu` component and only when the breakpoint size is `medium`.
 * `menuIsPinned` - A boolean flag indicating whether or not the menu is currently pinned. This flag is provied only to the `menu` component and only when the breakpoint size is `medium`.
-
-## Getting Started
```

Comment:
Why is this being removed?

Reply:
These were removed way back during the hackathon. I honestly don't remember why. 🤷‍♀️ I feel it was because the site now shows the raw example src for each example provided. I'll add back